easybuild/easyconfigs/b/bokeh/bokeh-3.2.2-foss-2023a.eb

```
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'bokeh'
+version = '3.2.2'
+
+homepage = 'https://github.com/bokeh/bokeh'
+description = "Statistical and novel interactive HTML plots for Python"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('meson-python', '0.13.2'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('matplotlib', '3.7.2'),
+    ('PyYAML', '6.0'),
+    ('Pillow-SIMD', '9.5.0'),
+    ('SciPy-bundle', '2023.07'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('tornado', '6.3.2'),
+    ('hatchling', '1.18.0'),  # fixes error "xyzservices>=2021.09.1, but you have 0.0.0"
+]
```

`hatchling` is only a build dependency:
```suggestion
builddependencies = [
    ('meson-python', '0.13.2'),
    ('hatchling', '1.18.0'),  # fixes error "xyzservices>=2021.09.1, but you have 0.0.0"
]

dependencies = [
    ('Python', '3.11.3'),
    ('matplotlib', '3.7.2'),
    ('PyYAML', '6.0'),
    ('Pillow-SIMD', '9.5.0'),
    ('SciPy-bundle', '2023.07'),
    ('Python-bundle-PyPI', '2023.06'),
    ('tornado', '6.3.2'),
]
```